### ISSUE TITLE
Run a11y tests during CI and error if any failures

### ISSUE THREAD

**Issue by davinotdavid** [thunderbird/services-ui#309]
Currently, we have the `addon-a11y` Storybook addon installed but it is tucked away / semi-hidden in the interface.

In an effort to enforce a11y standards, we should be running those in CI and failing PRs if it doesn't pass.
